css/index.css

```diff
@@ -1,3 +1,370 @@
+}
+
+h1 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+	margin-bottom: 1.5vh;
+	font-size: 72px;
+	text-align: center;
+}
+
+
+
+/* ------------------------ MOBILE ------------------------ */
+
+#entire-page {
+	width: 100%;
+}
+
+/* navigation and logo at the top of each page */
+.top-header-container{
+	width: 100%;
+}
+
+.top-header-container .top-header {
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+	align-items: center;
+}
+
+.top-header-container .top-header .top-header-nav {
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+	padding-bottom: 5%;
+}
+
+.top-header-container .top-header .top-header-logo {
+	width: 100%;
+	border-bottom: 1px solid #808080;
+	text-align: center;
+	padding: 4% 0;
+}
+
+.top-header-container .top-header .top-header-nav a {
+	width: 100%;
+	color: #b0b0b0;
+	border-bottom: 1px solid #808080;
+	font-size: 20px;
+	text-decoration: none;
+	text-align: center;
+	padding: 4% 0;
+}
+
+.top-header-container .top-header .top-header-nav a:hover {
+	color: black;
+}
+
+/* page container */
+.page-container {
+	display: flex;
+	flex-direction: column;
+	margin: 0 auto;
+	width: 100%;
+	padding: 0 5%;
+}
+
+/* header for the page container */
+.page-container .page-header {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	width: 100%;
+	padding: 10% 0;
+}
+
+.page-container .page-header .page-header-content{
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	width: 100%;
+}
+
+.page-container .page-header .page-header-img{
+	display: none;
+}
+
+.page-container .page-header .page-header-content button {
+	margin-top: 5%;
+	border: 1px solid black;
+	padding: 1% 10%;
+	font-size: 16px;
+}
+
+/* page divider */
+hr {
+	border: 0;
+	border-top: 2px solid black;
+	display: block;
+	width: 100%;
+}
+
+/* page content sections */
+.page-content {
+	width: 100%;
+	display: flex;
+	flex-direction: column;
+	padding-top; 
+}
+
+.page-content .page-content-section {
+	width: 100%;
+	padding: 5% 0;
+}
+
+.page-content .page-content-section .learn-more-button {
+	margin-top: 1.5vh;
+	border: 1px solid black;
+	background: white;
+	padding: 2% 5%;
+	border-radius: 5px;
+}
+
+.page-content-img {
+	display: block;
+	width: 100%;
+	height: auto;
+}
+
+.page-content .services-content {
+	border: 1px solid black;
+	background: #f4f3f4;
+	padding: 4% 4%;
+	margin-bottom: 5%;
+}
+
+.services-wrap {
+	flex-wrap: wrap;
+	margin-top: 5%;
+}
+
+/* contact */
+.contact {
+	padding: 5% 0;
+	display: flex;
+	flex-direction: column;
+}
+
+.block-address {
+	display:block;
+}
+
+/* footer */
+footer {
+	width: 100%;
+	text-align: center;
+	padding-bottom: 5%;
+}
+
+address {
+	padding-bottom: 1.5vh;
+}
+
+ul {
+	list-style-type: square;
+	margin: 0;
+	padding: 0;
+	list-style-position: inside;
+	margin-top: 1.5vh;
+}
+
+li {
+	padding-bottom: 1.5vh;
+}
+
+li:last-child {
+	padding-bottom: 0;
+}
+
+/* ------------------------ TABLET ------------------------ */
```

Comment:
I love how you have it separate your code with this style of comment, it's super helpful and really help me guide through your code with ease.